eng/pipelines/build.yml

```diff
@@ -20,177 +16,6 @@ resources:
       name: Azure/azure-sdk-tools
```

Comment:
This repository reference doesn't look to be used in the build steps anywhere and should be removed.

Reply:
removed